src/pages/PodcastListPage/PodcastListPage.spec.js

```diff
@@ -0,0 +1,91 @@
+import { render, screen } from '@testing-library/react'
+import PodcastListPage from './PodcastListPage'
+
+describe('PodcastListPage', () => {
+  it('renders two Podcast Cards', () => {
+    render(
+      <PodcastListPage
+        podcasts={[
+          {
+            id: 432618,
+            title: 'Murder On The Space Coast',
+            url:
+              'https://www.omnycontent.com/d/playlist/a858b0a5-e5e6-4a14-9717-a70b010facc1/6c32fe2a-7527-4559-b897-ab370147e602/61cbce74-5204-4af7-bf01-ab370147e615/podcast.rss',
+            originalUrl:
+              'https://www.omnycontent.com/d/playlist/a858b0a5-e5e6-4a14-9717-a70b010facc1/6c32fe2a-7527-4559-b897-ab370147e602/61cbce74-5204-4af7-bf01-ab370147e615/podcast.rss',
+            link: 'http://murderonthespacecoast.com/',
+            description:
+              "An unsympathetic victim, a prosecutor's conviction playbook, a missing mother of two and a tale of lie after lie after lie. Across four seasons, Murder on the Space Coast has explored each of these heartbreaking true-crime stories based in Central Florida.",
+            author: 'Gannett',
+            ownerName: 'Murder On The Space Coast',
+            image:
+              'https://www.omnycontent.com/d/playlist/a858b0a5-e5e6-4a14-9717-a70b010facc1/6c32fe2a-7527-4559-b897-ab370147e602/61cbce74-5204-4af7-bf01-ab370147e615/image.jpg?t=1578085538&size=Large',
```

Comment:
You should drastically simplify your example data – and also move it to an extra file, if neccessary.

Reply:
Please delete all properties you don't need.